docs/RUNNING-LOCALLY.md

```diff
@@ -31,6 +31,7 @@ The following MinGW packages are required:
 - `msys-unzip`
 - `msys-wget`
 - `msys-bash`
+- `msys-coreutils`
```

Comment:
`msys-coreutils` include the `zip` tool, which we're using in this PR. Of course the file where that is used is Linux specific, but since we proved that we can actually generate the AppDir from any OS (including Windows), I thought it was a good idea to include here anyway.